getGeocode.js

```diff
@@ -0,0 +1,80 @@
+const dotenv = require("dotenv");
+//note: this only works locally; in Lambda we use environment variables set manually
+dotenv.config();
+
+const fs = require("fs");
+const fetch = require("node-fetch");
+
+const { Client } = require("@googlemaps/google-maps-services-js");
+const geo = require("./geo.json");
+
+const getGeocode = async (name, street, zip) => {
+  const address = `${name},MA,${street},${zip}`;
+  const client = new Client();
+  try {
+    const resp = await client.geocode({
+      params: {
+        address,
+        key: process.env.GOOGLE_API_KEY,
+      },
+    });
+    return resp.data;
+  } catch (e) {
+    console.error(e.response.data);
+  }
+};
+
+const getLocationData = async () => {
```

Comment:
We shouldn't grab this twice; can you instead take advantage of the fact that we retrieve this in main.js?